Src/WitsmlExplorer.Frontend/components/Modals/ModalParts.tsx

```diff
@@ -0,0 +1,8 @@
+﻿export enum PropertiesModalMode {
+  New,
+  Edit
+}
+
+export const validText = (text: string): boolean => {
+  return text && text.length > 0;
```

Comment:
```
>>  "" && "".length > 0;
<-   ""
```
An empty string in this expression returns string and not boolean, I have not testet through the application though. It might be handled implicitly there? All yarn tests ran fine

Reply:
wrapping it like this should return a boolean
return !!(text && text.length> 0);

Reply:
I have not made any changes to the code here, just moved it into a separate class as it was duplicated several places.

Reply:
An empty string here will return false.

Reply:
https://stackoverflow.com/questions/11543735/why-the-result-of-booltrue-string-is-string-in-javascript